pandas-stubs/core/generic.pyi

```diff
@@ -163,15 +166,17 @@ class NDFrame(PandasObject, indexing.IndexingMixin):
     def to_sql(
         self,
         name: _str,
-        con,
+        con: Any,
         schema: _str | None = ...,
         if_exists: _str = ...,
```

Comment:
according to docs, `if_exists` is `Literal["fail", "replace", "append"]`

Reply:
Fixed.